rebar.config

```diff
@@ -1,7 +1,7 @@
 %%-*- mode: erlang -*-
 
 {deps, [
-        {lhttpc, "1.6.2"}
+        {hackney, {git, "git@github.com:benoitc/hackney.git", {tag, "1.17.4"}}}
```

Comment:
Shouldn't this be our own fork? Or just hex would be even better. Then we'll override it in core.

Reply:
since this is a public repo I would like to use the external original versions. It will be overwritten for kivra_core and ek_mm, but it is weird since our branch of hackney has diverged from the original fork.